src/modules/jcms_rest/src/Plugin/rest/resource/PingRestResource.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Drupal\jcms_rest\Plugin\rest\resource;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Provides a ping for smoke tests to call.
+ *
+ * @RestResource(
+ *   id = "ping_rest_resource",
+ *   label = @Translation("Ping rest resource"),
+ *   uri_paths = {
+ *     "canonical" = "/ping"
+ *   }
+ * )
+ */
+class PingRestResource extends AbstractRestResourceBase {
```

Comment:
There must be a simpler way to add a controller than extending whatever `AbstractRestResourceBase` is?